protonmail/protonmail.go

```diff
@@ -154,7 +154,7 @@ func (c *Client) do(req *http.Request) (*http.Response, error) {
 
 func (c *Client) doJSON(req *http.Request, respData interface{}) error {
 	req.Header.Set("Accept", "application/json")
-
+	req.Header.Set("User-Agent","Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36")
```

Comment:
Can we move this to `Client.do`? That way it'll be applied to all HTTP requests.

(I'd also slightly prefer to pick a user-agent describing an open-source browser on an open-source OS.)